protocol/message.go

```diff
@@ -1,87 +1,152 @@
+// Defines the message formats of the CONIKS protocols
```

Comment:
And just in case this isn't intentional: this comment won't show up in the rendered godoc documentation.